scripts/queue_report.py

```
@@ -0,0 +1,141 @@
+        print("yet seen two cycles on one level -- nothing can be differenced")
+        print("until it has. If a run is in flight, wait one cycle.")
+        return 1
+
+    span_min = (max(r["ts"] for r in rows) - min(r["ts"] for r in rows)) / 60.0
+    by_market: dict[str, list[dict]] = {}
+    for r in rows:
+        by_market.setdefault(r["market_slug"] or "?", []).append(r)
+
+    print(f"queue marks: {len(rows)} rows over {span_min:.1f} minutes, "
+          f"{len(by_market)} markets")
+    print(f"DIRECTIONAL at this span -- enough to separate 'under an hour' "
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)